### PR TITLE
UHM-4537 Lab Results not showing on patient results view

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/UHM-4537

Screenshot from 2020-05-15 14-39-58

### CODE REVIEW

File: app/js/components/LabResultsList.jsx [hunk 1]
```diff
@@ -147,17 +151,37 @@ export class LabResultsList extends PureComponent {
...
+          navigate={this.handleShowLabTrendsPage}
+          locale={this.props.locale}
+        />
+      ),
+      className: `lab-results-list-cell-${columnName
+        .replace(" ", "-")
+        .toLocaleLowerCase()}`,
+      headerClassName: `lab-results-list-column-header lab-results-list-header-${columnName
+        .replace(" ", "-")
+        .toLocaleLowerCase()}`,
```
Comment (brandones) [inline app/js/components/LabResultsList.jsx:184]: No code change, just autoformat.

**Comment by brandones** [comment]
This just fixes my previous change; see https://github.com/openmrs/openmrs-owa-labworkflow/commit/865bdc7978067be1a0952f1df36749bab75f21ec#diff-81f06989ded9a69fc3980d4781d7cb34 line 46

File: app/js/components/LabResultsList.jsx [hunk 2]
```diff
@@ -43,7 +43,11 @@ const Cell = ({
   if (columnName === 'RESULT') {
     return (
       <div className="table_cell result">
-        {getConceptShortName(value.value, locale)}
+        <ConceptDisplay
+          conceptUUID={value.concept.uuid}
+          type="result"
+          value={value.value ? getConceptShortName(value.value, locale) : null}
+        />
```
Comment (mogoodrich) [inline app/js/components/LabResultsList.jsx:50]: This doesn't seem right.  And it looks like I missed the error in the last PR we worked on as well that is what broke this,

There are a couple keys here:
* The top-level "value" variable is poorly named which makes things confusing, it's really an obs. We should have named that variable "obs".
* The obs value could be a concept or value (for numeric obs)

The ConceptDisplay tag takes the uuid of the question concept, and the a value that is either a string or a numeric.  

So I think the logic should be more like this pseudo code:

// to make things clearer
let obs = value

// conceptUUID to past to Concept Display (uuid of the "question" concept)
conceptUUID = obs.concept.uuid

// for the value
if (obs.value is a concept) then getConceptShortName(obs.value, locale)
else if (obs.value is a numeric) then obs.value

**Comment by brandones** [comment]
Thanks for the detailed analysis @mogoodrich . Should be fixed.

Screenshot from 2020-05-18 15-39-26

**Pull request merged**